pkgs/applications/misc/gpx/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "gpx-${version}";
+  version = "2.5.2";
+
+  buildInputs = [ autoreconfHook ];
```

Comment:
Use `nativeBuildInputs` instead.

Reply:
Sorry, I was following the example on the wiki.

Reply:
Got a link to it, so we can fix it?

Reply:
In https://nixos.wiki/wiki/Packaging_Software the code sample is fine, but the link to a file that I based gpx off of is pinned to a version of nixpkgs using buildInputs.

Reply:
Checked, this still isn't fixed